### PR TITLE
Fix localized builds on xplat

### PR DESCRIPTION
**Pull request by cdmihai**

### CODE REVIEW

**Comment by cdmihai** [comment]
@dotnet-bot test OSX Build for CoreCLR please

File: cibuild.cmd [hunk 1]
```diff
@@ -58,14 +59,19 @@ if /i "%HOST%"=="CoreCLR" (
...
+
 :: Restore build tools
 call %~dp0init-tools.cmd
 
 echo.
 echo ** Rebuilding MSBuild with downloaded binaries
 
 set MSBUILDLOGPATH=%~dp0msbuild_bootstrap_build.log
-call "%~dp0build.cmd" /t:Rebuild /p:Configuration=%BUILD_CONFIGURATION% /p:"SkipBuildPackages=true"
+call "%~dp0build.cmd" /t:Rebuild /p:Configuration=%BUILD_CONFIGURATION% /p:"SkipBuildPackages=true" %LOCALIZED_BUILD_ARGUMENT%
```
Comment (rainersigwald) [inline cibuild.cmd]: Does the first bootstrapping build need to be localized? Seems like it's a final-build-only kind of thing.

Of course, we don't have good logic for the `--bootstrap-only` case.

**Comment by rainersigwald** [comment]
@dotnet-bot test OSX Build for CoreCLR please

**Comment by cdmihai** [comment]
@dotnet-bot test OSX Build for CoreCLR please

File: cibuild.cmd (hunk 1 above)
Comment (cdmihai) [inline cibuild.cmd]: I'd keep it on. The purpose of the badly named flag is for dev experience. You don't want to do 2 builds every time. But I'd rather force localized builds every time so it's tested a lot :)

Comment (cdmihai) [inline cibuild.cmd]: However, you do get dirty xlf files each time. Okay, my next thing is to separate xlf synchronization, just for the experience of it :)

**Comment by cdmihai** [comment]
@dotnet-bot test OSX Build for CoreCLR please

**Pull request merged**